packages/lesswrong/components/sunshineDashboard/SunshineNewUsersInfo.tsx

```diff
@@ -127,6 +135,7 @@ const SunshineNewUsersInfo = ({ user, classes, updateUser }: {
   const currentUser = useCurrentUser();
 
   const [notes, setNotes] = useState(user.sunshineNotes || "")
+  const [contentSort, setContentSort] = useState("baseScore")
```

Comment:
Should be safe to narrow down the `useState` type to `'baseScore' | 'postedAt'`, given its usage - normally I'd give it an explicit alias, but afaict codebase convention seems to mostly point at "inline types unless they're used in multiple places"